task/deprecated-image-check/0.1/deprecated-image-check.yaml

```diff
@@ -33,6 +35,7 @@ spec:
       image: registry.access.redhat.com/ubi8/ubi
       script: |
         #!/usr/bin/env bash
+        echo "$(params.BASE_IMAGES_DIGESTS)"
         echo "Querying Pyxis for $(params.IMAGE_REPOSITORY)..."
         http_code=$(curl -s -k -o $(workspaces.sanity-ws.path)/repository_data.json -w '%{http_code}' "https://catalog.redhat.com/api/containers/v1/repositories/registry/$(params.IMAGE_REGISTRY)/repository/$(params.IMAGE_REPOSITORY)")
```

Comment:
What if the user copies the image from the Red Hat registry into their own registry? Do we want the check to fail in such caes? It would be great if we could query pyxis by digest regardless of which repo/registry the image resides in. Not sure if the existing Pyxis API allows that.

Reply:
good point. I will capture this in a separate Jira.

Reply:
BTW, I asked the Pyxis maintainers a few weeks  back and they pointed me to: `/v1/images?filter=image_id==sha256:<digest>`

As you said, the implementation on this PR has limited focus, and that's ok. Just sharing some info that could be useful in the future.